src/tests/generators/api.ts

```diff
@@ -84,23 +84,28 @@ type HandlerRequest<T> = DefaultBodyType & {
 };
 
 export const createSoapAPIInterceptorWithError = <RequestParamsType>(
-	apiAction: string
+	apiAction: string,
+	forceFail = false
```

Comment:
The parameter is altering the contract of the function. The returned promise should resolve as soon as the API is called